packages/smooth_app/lib/cards/data_cards/image_upload_card.dart

```diff
@@ -69,8 +71,10 @@ class _ImageUploadCardState extends State<ImageUploadCard> {
             User(userId: 'smoothie-app', password: 'strawberrybanana');
```

Comment:
@teolemon this is one of two hardcoded app users, this even added two test (non food) products to the DB, when I remeber right we don't even need them anymore since we do not longer need a user to add a photo, right?

Reply:
- "we do not longer need a user to add a photo" ?
  - We have never formally needed a user for photos (it's attributed to openfoodfacts-contributors). I believe @stephanegigandet created a specific user to see how efficient Smoothie was at gathering photos. 
- We still have the case where the user is not logged in, and where we could use that user